### PR TITLE
Add New Port Mire

### PR DESCRIPTION
**Pull request by MadShmupper**
Add New Port Mire

# New Port for { Mire }

## Game Information
- **Title**: { Mire }
- **URL**: { https://bagelcollie.itch.io/mire }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/mire/Mire.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# PORTMASTER: mire.zip, Mire.sh
```
Comment (Cebion) [inline ports/mire/Mire.sh:2]: This is autogenerated by PortMaster. Not needed

File: ports/mire/Mire.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+export PORT_32BIT="Y"
+
+get_controls
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+$ESUDO chmod 666 /dev/tty0
+
+GAMEDIR="/$directory/ports/mire"
+
+export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs"
```
Comment (Cebion) [inline ports/mire/Mire.sh:27]: wrong LD_LIBRARY_PATH, you fixed it in your other releases =)

File: ports/mire/cover.png
Comment (Cebion) [inline ports/mire/cover.png:1]: Cover should not be just a screenshot!
If you only have a screenshot don't use a cover.
Instead use screenshot.png in your gameinfo.xml

File: ports/mire/port.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        "image": null,
+        "rtr": true,
+        "exp": false,
+        "runtime": null,
+        "reqs": [],
+        "arch": [
+            "armhf"
+        ]
+    },
+    "status": {
```
Comment (Cebion) [inline ports/mire/port.json:31]: You added a port.json from an installed copy. None of the bottom fields should be in it =)

**Review by Cebion — CHANGES_REQUESTED** [review]

**Review by Cebion — APPROVED** [review]

**Review by Slayer366 — APPROVED** [review]

**Pull request merged**